### PR TITLE
HTTP stress docker fix

### PR DESCRIPTION
**Pull request by ManickaP**
Fixed docker files not using currently built runtime for http stress test.

### CODE REVIEW

File: src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ARG SDK_BASE_IMAGE=mcr.microsoft.com/dotnet/nightly/sdk:5.0-buster-slim
 FROM $SDK_BASE_IMAGE
 
-RUN echo "DOTNET_SDK_VERSION="$env:DOTNET_SDK_VERSION
+RUN echo "DOTNET_SDK_VERSION="$DOTNET_SDK_VERSION
```
Comment (ManickaP) [inline src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile:4]: I think we should get rid of these prints completely in all dockerfiles.

Comment (alnikola) [reply src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile:4]: It was suggested to add by .Net Engineering Services team to make investigations easier.

**Review by alnikola — APPROVED** [review]
LGTM.

**Comment by ManickaP** [comment]
There might be a follow up, I'm going through the pipeline logs on CI and I'm seeing some discrepancies between windows and linux docker outputs.

**Pull request merged**